lychee-lib/src/filter/mod.rs

```diff
@@ -120,8 +128,12 @@ impl Filter {
     ///    - When the excludes rules matches the URI, it's *explicitly excluded*.
     #[must_use]
     pub fn is_excluded(&self, uri: &Uri) -> bool {
-        // Skip mail address, specific IP, and scheme
-        if self.is_mail_excluded(uri) || self.is_ip_excluded(uri) || self.is_scheme_excluded(uri) {
+        // Skip mail address, specific IP, specific host and scheme
+        if self.is_mail_excluded(uri)
+            || self.is_ip_excluded(uri)
+            || self.is_host_excluded(uri)
+            || self.is_scheme_excluded(uri)
+        {
```

Comment:
Using multiple lines was forced by CI lint. Not sure why it wasn't before, probably 3 -> 4 conditions was the break point or so.

Reply:
Yeah, it got a little messy. 😆